src/components/SidePanel/index.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import styles from './styles.module.scss';
+
+import SidePanel from '.';
+
+describe(__filename, () => {
+  const render = ({
+    children = <p />,
+    className = 'class-name',
+    isExpanded = true,
+    onClick = jest.fn(),
+    toggleLeft = false,
+  } = {}) => {
+    return shallow(
+      <SidePanel
+        className={className}
+        isExpanded={isExpanded}
+        onClick={onClick}
+        toggleLeft={toggleLeft}
+      >
+        {children}
+      </SidePanel>,
+    );
+  };
+
+  it('marks the top-level as expanded when it is expanded', () => {
+    const root = render({ isExpanded: true });
+
+    expect(root).toHaveProp('aria-expanded', 'true');
+  });
+
+  it('marks the top-level as collapsed when it is collapsed', () => {
+    const root = render({ isExpanded: false });
+
+    expect(root).toHaveProp('aria-expanded', 'false');
+  });
+
+  it('sets a className when collapsed', () => {
+    const root = render({ isExpanded: false });
+
+    expect(root).toHaveClassName(`.${styles.isCollapsed}`);
```

Comment:
Since this is testing the same logic as `aria-expanded`, I think you could combine this assertion into that test.